### ISSUE TITLE
How to render Z-depth map?

### ISSUE THREAD

**Issue by josai** [HackerPoet/PySpace#11]
Is it possible to render out just a Z-depth map for compositing in software later?

**Comment by HackerPoet** [comment]
Yes, I just pushed in some changes including a proper update of the depth map.  The depth is in the range 0.0 to 1.0 where you can multiply by MAX_DIST to get the true depth.
